.gitignore

```diff
@@ -3,3 +3,4 @@
 *#
 .DS_Store
 .env
+queue_listener/__pycache__
```

Comment:
is it worth just ignoring this across the board? I can't think of any place we'd want to commit it